### PR TITLE
feat: Sync labeled data from Argila back to phospho

### PR DESCRIPTION
**Pull request by flamschou**

### CODE REVIEW

File: backend/app/api/platform/endpoints/integrations.py [hunk 1]
```diff
@@ -94,3 +102,77 @@ async def post_postgresql_push(
...
+            status_code=400, detail="The dataset name does not exist for this project."
+        )
+    argilla_dataset = await pull_dataset_from_argilla(request)
+    for record in argilla_dataset.records:
+        if len(record.responses) > 0:
+            logger.debug(f"Argilla dataset: {record}")
+    return {"status": "ok"}
+
+
+@router.post("/argilla/names")
```
Comment (oulianov) [inline backend/app/api/platform/endpoints/integrations.py:144]: Please rename to `argilla/datasets`

File: backend/app/services/integrations/argilla.py [hunk 2]
```diff
@@ -73,6 +74,54 @@ def dataset_name_is_valid(dataset_name: str, workspace_id: str) -> bool:
         return False
 
 
+def dataset_name_exists(dataset_name: str, workspace_id: str, project_id: str) -> bool:
+    """
+    For now, checks if the name does not already exist in the workspace
+    """
+    if len(dataset_name) == 0:
+        return False
```
Comment (oulianov) [inline backend/app/services/integrations/argilla.py:82]: not needed

File: backend/app/api/platform/endpoints/integrations.py (hunk 1 above)
Comment (flamschou) [reply backend/app/api/platform/endpoints/integrations.py:144]: argilla/datasets is already the endpoint used to create the argilla dataset.

**Review by oulianov — APPROVED** [review]
good !

**Pull request merged**